conf/local.conf.nonostree.append

```diff
@@ -0,0 +1,19 @@
+
+# Store systemd logs in persistent storage
+#
+# It greatly helps diagnosing issues on testing devices but should be
+# carefully weighted against file system usage and flash device wear for
+# production systems. Please refer to systemd's docs for more details
+IMAGE_INSTALL_append += " systemd-journald-persistent"
+
+# Set resource limits for aktualizr service
+#
+# Uncomment these lines to change the default parameters.
+#
+#RESOURCE_CPU_WEIGHT_pn-aktualizr = "100"
+#RESOURCE_MEMORY_HIGH_pn-aktualizr = "100M"
+#RESOURCE_MEMORY_MAX_pn-aktualizr = "80%"
+IMAGE_INSTALL_append += " aktualizr-resource-control"
+
+# The default sota hardware id. 
+#SOTA_HARDWARE_ID ?= "${MACHINE}-non-ostree"
```

Comment:
Is there a need to suggest changing the `SOTA_HARDWARE_ID` here? The only config appends don't do it and I don't see why we need it here. In fact, I see no problem with using the base and systemd appends as they are. Or am I overlooking something?